lib/JSCRunner.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const fs = require('fs');
+const temp = require('temp');
+const inception = require('./inception');
+const ConsoleRunner = require('./ConsoleRunner');
+
+const errorRe = /^(?:.*?): (\w+)(?:: (.*))$/m;
+
+// JSC stack frame format:
+// StackFrames: StackFrame+
+// StackFrame: FunctionFrame | NativeFrame | EvalMarker
+// FunctionFrame: (FunctionName`@`)?SourceInfo
+// NativeFrame: (FunctionName`@`)?`[native code]`
+// FunctionName: .*
+// SourceInfo: File`:`Line`:`Column
+// File: .*
+// Line: \d+
+// Column: \d+
+// EvalMarker: `eval code`
+const frameRe = /(?:(.*)@)?(\[native code\]|(?:(.*):(\d+):(\d+)))/;
+
+const newGlobalTempFile = temp.path({ suffix: '.js' });
+fs.writeFileSync(newGlobalTempFile, 'arguments[0](this);');
```

Comment:
The temporary file nonsense should be replaced with a simple static file which contains the single line `arguments[0](this);`. Is it okay to create this file under the 'runtimes' directory, or do you prefer a different directory?

The reason why we need that additional file:
JSC doesn't provide a simple shell function to create a new global object, so we're stuck with the built-in [run function](https://github.com/WebKit/webkit/blob/25d38212abad3c05df3bc3aaa607e5159a361427/Source/JavaScriptCore/jsc.cpp#L1243). The `run()` function requires [a file to execute](https://github.com/WebKit/webkit/blob/25d38212abad3c05df3bc3aaa607e5159a361427/Source/JavaScriptCore/jsc.cpp#L1247), but thankfully it's also possible to [pass arguments](https://github.com/WebKit/webkit/blob/25d38212abad3c05df3bc3aaa607e5159a361427/Source/JavaScriptCore/jsc.cpp#L1252-L1256) to the new global. That means we can make `arguments[0]` a callback function to pass the new global to our `createRealm` method. :tada: